### PR TITLE
Creep fluff completely rewritten

### PR DESCRIPTION
**Pull request by tralezab**
I wanted a subtle side antagonist that made actions as sacred as hugging or taking photos to induce paranoia, and I got some kind of "roleplay antagonist" that tiptoes rule 8 and murderbones. I wanted to squash at least one of those issues, and I have a fix for the murderboning that only removes the bad mood if you have completed the objectives before murdering the target.

but that's not in this pr, lets get into what this does with a couple examples

Antagonist name Creep > Obsessed - Makes more sense with the new greet, but essentially I need to distance this antag from the term "Creep"
Erotomania > Psychotic Schizophrenia - Supports the idea of voices controlling what you do, not yourself.
I'm so close to my obsession and I never want this to end. > The voices are happy! I feel free again!

and finally, the greet text. here's the new one:
	to_chat(owner, "<span class='boldannounce'>You are the Obsessed!</span>")
	to_chat(owner, "<B>These voices will only be sated by the horrible acts of murder and general evils.</B>")
	to_chat(owner, "<B>I don't know their connection, but the Voices compel me to stalk [trauma.obsession], to drive them crazy with paranoia.</B>")
	to_chat(owner, "<B>They will try to tear my mind apart if I avoid these tasks. I must free myself from the foreign presence in my head controlling me.</B>")
	to_chat(owner, "<span class='boldannounce'>You are NOT a roleplay antagonist. This does not let you tiptoe the lines of rule 8 like some kind of Creep. Control yourself.</span>")

hopefully this will explain the Obsessed to be more of what I wanted out of it, instead of "Her gazongas were MASSIVE"

### CODE REVIEW

File: code/modules/antagonists/creep/creep.dm [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /datum/antagonist/creep
```
Comment (tralezab) [inline code/modules/antagonists/creep/creep.dm]: I will have to change the pathing on this, I am aware

**Comment by Shadowflame909** [comment]
Yeah, now it looks more like a side antagonist.

Although, I hope there'd be some flavor text that'd go against people wanting to rush against medbay to cure it. With its new flavor-text.

Maybe something like, "If I alert people about this, the voices will get mad at me..."

**Comment by tralezab** [comment]
I want to avoid any connection to the voices because any connection just makes the user creepy again.

Any suggestions for better flavor or even just a different antag name would be great, you know i'm not great at this whole flavor thing from the fugitives or from some guy on the forum interpreting my old creep greet as "You are the rapist!"

**Comment by tralezab** [comment]
Maybe something like i'm trapped in my mind and the voices have completely taken over, meaning all of the actions are "the voice" in control. that was it would solve the issues of avoiding getting the trauma cured

**Comment by tralezab** [comment]
My next idea is a "Vengeful spirit" that hates the obsession, and has possessed the antagonist to kill the obsession but not before giving some clues it's coming

**Comment by nervere** [comment]
This quells my concerns regarding rule 8 for the creep, I look forward to see what you have in store to fix the murderboning issue.

**Comment by Mickyan** [comment]
I don't think you should mention specific rules in flavor text, else it's going to get very confusing for downstreams

if people really need a refresher that the rules exist you could ask an admin to put a notice in the MOTD for a while

**Comment by tralezab** [comment]
Well, the issue with murderboning right now is that the debilitating mood penalties go away if the target is dead, so someone who wants to murderbone can simply kill their obsession and then kill everyone else. Making the mood changes go away only after the obsession is dead and the objectives that aren't checked roundend (stealing) are complete. That way, the obsession has to greentext before they can even start terrorizing the station

**Comment by Shadowflame909** [comment]
Is this not also a fix to the murderboning issue?

I mean the voice isn't saying, "Go on a killing spree" It's literally just an insane guy, who needs to be near this other person to stop being taunted by voices in his head.

I mean if some nerd starts dragging him away from this other person, then he'll have some justifiable escalatory issue with that. But all in all, it'd be a stretch to try to murderbone, since the only reason your "obsessed" in the first place, is because of an outside force.

The murder portion would more or less fall on escalation.

**Comment by tralezab** [comment]
@Mickyan i'll consider that in my next rewrite of the greet, thank you

@Shadowflame909 We'll see what the final product turns up but it's not too far of a stretch that a vengeful possession may move on to other members of the station or schizophrenia leads to thinking others are accomplices of the obsession

**Comment by Shadowflame909** [comment]
Yeah, but now it seems like you'd have to agitate the side-antagonist in some way, by preventing them from getting their objective done. For them to escalate upon you.

I don't see this side antagonist just plasma flooding or killing a large number of people for no reason, because they themselves are not the real antagonist. It's just more or less torturing them until they do what the voice wants.

**Comment by tralezab** [comment]
How about we get this merged, and then see if people still murderbone excessively with it so we can know if I gotta throttle the killing

**Comment by hexmaniacosanna** [comment]
I like trying to make it less murderbone, but the issue you're always gonna have is an antag is gonna be treated as both a free murder card for them and a free validhunt card by  the crew. So it's difficult to truly remove the issue. I really like creep conceptually, and I've thought about this before, but adding additional objectives and not ALWAYS having them end in murder would up paranoia and reduce immediate validhunt. Say an objective to maroon the obsession with you rather than kill them. I don't know what your exact vision is, but I expect creep to be a more paranoia focused side-antag, which isn't true if you can always see them coming a mile away, and always know they are going to kill people.

**Comment by optimumtact** [comment]
`to_chat(owner, "These voices will only be sated by the horrible acts of murder and general evils.")`

maybe try some phrasing that is less related to murder

**Comment by OverlyGeneric** [comment]
Let's just be blunt with the name. Stalker is a synonym for a hunter and also a term for someone that harasses someone with unwanted or obsessive intention.

 

> `to_chat(owner, "These voices will only be sated by the horrible acts of murder and general evils.")`
> 
> maybe try some phrasing that is less related to murder

https://www.youtube.com/watch?v=Ag1o3koTLWM

**Comment by tralezab** [comment]
if i said mass murder it would be worrying but creeps get objectives to kill multiple people so i think it's just being truthful

**Comment by tralezab** [comment]
@OverlyGeneric It's a first draft be nice

**Comment by OverlyGeneric** [comment]
> @OverlyGeneric It's a first draft be nice :(

@tralezab First draft or not. You got mildly toasted by a fruit and baked by a homecooked meal.

**Comment by SpaceManiac** [comment]
> You are NOT a roleplay antagonist. This does not let you tiptoe the lines of rule 8 like some kind of Creep. Control yourself.

I know the old text did this too, but if you're changing this, don't reference the rule by number. The rules are controlled externally, Rule 8 could become something else at any time.

**Comment by tralezab** [comment]
@subject217 I agree, that rapist post on the policy discussion is why I'm never making a forum account jesus. I tried to avoid making it at all about roleplaying by giving it very explicit objectives, the opposite of the revenant's freeform roleplay objectives

File: code/modules/antagonists/creep/creep.dm [hunk 2]
```diff
@@ -23,10 +23,11 @@
...
 	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/creepalert.ogg', 100, FALSE, pressure_affected = FALSE)
-	to_chat(owner, "<span class='boldannounce'>You are the Creep!</span>")
-	to_chat(owner, "<B>They would call it an obsession. They would call you crazy, because they don't understand your unrequited love.<br>All you know is that you love [trauma.obsession]. And you. will. show them.</B>")
-	to_chat(owner, "<B>I will surely go insane if I don't spend enough time around [trauma.obsession], but when i'm near them too long it gets too difficult to speak properly, making me look like a CREEP!</B>")
-	to_chat(owner, "<span class='boldannounce'>The gods would like to remind you that this role, as with all other antags, does not allow you to break ANY server rules, especially Rule 8 (These rules being listed from the \"Rules\" button at the top right of your mind's screen). Feel free to murder and pillage just like any other antag, though.</span>")
+	to_chat(owner, "<span class='boldannounce'>You are the Obsessed!</span>")
+	to_chat(owner, "<B>These voices will can only be sated by the horrible acts of murder and general evils.</B>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/antagonists/creep/creep.dm]:
```suggestion
	to_chat(owner, "<B>These voices can only be sated by the horrible acts of murder and general evils.</B>")
```

File: code/modules/antagonists/creep/creep.dm [hunk 3]
```diff
@@ -23,10 +23,11 @@
...
+	to_chat(owner, "<B>I don't know their connection, but the Voices compel me to stalk [trauma.obsession], to drive them crazy with paranoia.</B>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/antagonists/creep/creep.dm]:
```suggestion
	to_chat(owner, "<B>You don't know their connection, but the Voices compel you to stalk [trauma.obsession], forcing them into a state of constant paranoia.</B>")
```

File: code/modules/antagonists/creep/creep.dm [hunk 4]
```diff
@@ -23,10 +23,11 @@
...
+	to_chat(owner, "<B>They will try to tear my mind apart if I avoid these tasks. I must free myself from the foreign presence in my head controlling me.</B>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/antagonists/creep/creep.dm]:
```suggestion
	to_chat(owner, "<B>The Voices will retaliate if you fail to complete your tasks. You must free yourself from the foreign presence in your head controlling you.</B>")
```

File: code/modules/antagonists/creep/creep.dm [hunk 5]
```diff
@@ -23,10 +23,11 @@
...
+	to_chat(owner, "<span class='boldannounce'>You are NOT a roleplay antagonist. This does not let you tiptoe the lines of rule 8 like some kind of Creep. Control yourself.</span>")//ironic if you know the history of the antag
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/antagonists/creep/creep.dm]:
```suggestion
	to_chat(owner, "<span class='boldannounce'>This role does NOT let you tiptoe the lines of creepy behavior viewed as excessive per the rules.</span>")//ironic if you know the history of the antag
```

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
convert me's to you's OR give them a unique span so the shift is more apparent.

I've done the easier of the two for you below :)

**Comment by tralezab** [comment]
Good to keep in mind but I'm rewriting that today as I dont like the idea of the voices, there are already 2 schizophrenia traumas

**Comment by ExcessiveUseOfCobblestone** [comment]
kk

**Comment by tralezab** [comment]
nevermind i like yours more i'm sticking with it

**Comment by tralezab** [comment]
alright it compiles, not tested

File: code/modules/antagonists/creep/creep.dm (hunk 1 above)
Comment (tralezab) [reply code/modules/antagonists/creep/creep.dm]: done

**Comment by ExcessiveUseOfCobblestone** [comment]
lol "mine" is just yours with a different subject, don't have to be modest!

**Comment by ExcessiveUseOfCobblestone** [comment]
just uh let me know when it's ready (tested)

**Comment by tralezab** [comment]
image

image

seemingly works, but let me remove the MUST free line since it doesn't work well in game and remove the heart eyes the moodlet gets, it doesn't make sense anymore

**Comment by tralezab** [comment]
@ExcessiveUseOfCobblestone tested (somewhat, it's hard to do with two clients) and ready

**Pull request merged**